smart Ease.js

```diff
@@ -0,0 +1,18 @@
+// Fullname Function
+function getFullname(firstname, surname, useFormalName = false, gender = "male") {
+    if (!firstname || !surname) {
+        return "Name is incomplete";
+    }
+    let fullname = firstname + " " + surname;
```

Comment:
Could be cleaner with a string literal.
```suggestion
    let fullname = `${firstname}  ${surname}`;
```